### PR TITLE
Features/psp 6825

### PR DESCRIPTION
**Pull request by FuriousLlama**

### CODE REVIEW

File: source/backend/entities/PimsBaseContext.cs [hunk 1]
```diff
@@ -275,7 +275,6 @@ public PimsBaseContext(DbContextOptions<PimsBaseContext> options)
         public virtual DbSet<PimsYearlyFinancialCode> PimsYearlyFinancialCodes { get; set; }
         public virtual DbSet<PimsYearlyFinancialCodeHist> PimsYearlyFinancialCodeHists { get; set; }
         public virtual DbSet<PimsxTableDefinition> PimsxTableDefinitions { get; set; }
-        public virtual DbSet<TmpPimsPropPropAdjacentLandType> TmpPimsPropPropAdjacentLandTypes { get; set; }
```
Comment (asanchezr) [inline source/backend/entities/PimsBaseContext.cs:278]: is this related to this PR??

File: source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.test.tsx [hunk 2]
```diff
@@ -162,27 +162,6 @@ describe('AcquisitionContainer component', () => {
...
-          id: 1,
-        },
-        activityInstanceProperties: [{}],
-      },
-    ]);
-    await act(async () => viewProps.onShowPropertySelector());
-    const canRemoveResponse = await act(() => viewProps.canRemove(1));
-    expect(canRemoveResponse).toBe(false);
-  });
-
```
Comment (asanchezr) [inline source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.test.tsx:185]: why delete an existing (passing) test?

File: source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.tsx [hunk 3]
```diff
@@ -212,12 +212,7 @@ export const AcquisitionContainer: React.FunctionComponent<IAcquisitionContainer
   };
 
   const canRemove = async (propertyId: number) => {
-    const fileProperties = await retrieveAcquisitionFileProperties(acquisitionFileId);
-    const fp = fileProperties?.find(fp => fp.property?.id === propertyId);
-    return (
-      fp?.activityInstanceProperties?.length === undefined ||
-      fp?.activityInstanceProperties?.length === 0
-    );
+    return true;
```
Comment (asanchezr) [inline source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.tsx:215]: could you please explain why we removed the existing logic?

File: source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/PropertyContactList.tsx [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+          </Row>
+        );
+      },
+    },
+  ];
+
+  return columns;
+}
+
+const FilePropertiesTable: React.FunctionComponent<IPropertyContactListProps> = ({
```
Comment (asanchezr) [inline source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/PropertyContactList.tsx:126]: nit: I'd prefer to name the component the same as the filename - PropertyContactList

**Review by asanchezr — APPROVED** [review]

File: source/backend/entities/PimsBaseContext.cs [hunk 5]
```diff
@@ -3196,8 +3195,6 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 entity.Property(e => e.InterestHolderHistId).HasDefaultValueSql("(NEXT VALUE FOR [PIMS_INTEREST_HOLDER_H_ID_SEQ])");
 
                 entity.Property(e => e.EffectiveDateHist).HasDefaultValueSql("(getutcdate())");
-
-                entity.Property(e => e.InterestHolderTypeCode).HasDefaultValueSql("(N'INTHLDR')");
```
Comment (asanchezr) [inline source/backend/entities/PimsBaseContext.cs:3200]: seems unrelated, not sure

File: source/backend/api/Models/Concepts/Property/PropertyModel.cs [hunk 6]
```diff
@@ -222,6 +222,11 @@ public class PropertyModel : BaseModel
         /// </summary>
         public string GeneralLocation { get; set; }
 
+        /// <summary>
+        /// get/set - Property contacts.
+        /// </summary>
+        public List<PropertyContactModel> PropertyContacts { get; set; }
+
```
Comment (asanchezr) [inline source/backend/api/Models/Concepts/Property/PropertyModel.cs:229]: question - I did not see the corresponding `propertyContacts` array on the frontend API model. Is this intended?

File: source/backend/api/Models/Concepts/Property/PropertyContactModel.cs [hunk 7]
```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+
+namespace Pims.Api.Models.Concepts
+{
+    /// <summary>
+    /// PropertyContactModel class, provides a model to represent the property whether Land or Building.
+    /// </summary>
```
Comment (asanchezr) [inline source/backend/api/Models/Concepts/Property/PropertyContactModel.cs:8]: please update documentation

File: source/backend/api/Models/Concepts/Property/PropertyModel.cs [hunk 8]
```diff
@@ -222,6 +222,11 @@ public class PropertyModel : BaseModel
         /// </summary>
         public string GeneralLocation { get; set; }
 
+        /// <summary>
+        /// get/set - Property contacts.
+        /// </summary>
+        public List<PropertyContactModel> PropertyContacts { get; set; }
```
Comment (asanchezr) [inline source/backend/api/Models/Concepts/Property/PropertyModel.cs:228]: nit: the other "list" fields are declared via the interface **IList** - I'd be inclined to use the interface here (instead of the concrete implementation **List**) to keep it consistent

File: source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.tsx (hunk 3 above)
Comment (FuriousLlama) [reply source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.tsx:215]: There are not activity instances anymore. This should have been removed once we removed the activities.

File: source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.test.tsx (hunk 2 above)
Comment (FuriousLlama) [reply source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.test.tsx:185]: There is no longer a functional difference. See the answer below

File: source/backend/api/Models/Concepts/Property/PropertyModel.cs (hunk 6 above)
Comment (FuriousLlama) [reply source/backend/api/Models/Concepts/Property/PropertyModel.cs:229]: This is a minimun set of changes. Once we get the scaffold the models will need to be aligned

File: source/backend/api/Models/Concepts/Property/PropertyModel.cs (hunk 8 above)
Comment (FuriousLlama) [reply source/backend/api/Models/Concepts/Property/PropertyModel.cs:228]: sure
